charts/latest-v2/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -1,3 +1,33 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csi-azuredisk-controller-config
```

Comment:
We should use the name `{{ .Values.controller.name }}-config` for the `ConfigMap` name to allow for side-by-side installs in the future.